.github/workflows/python-package.yml

```diff
@@ -37,3 +37,13 @@ jobs:
         export PYTHONPATH=./pymgm:$PYTHONPATH
         pytest ./pymgm/test/
```

Comment:
Run export PYTHONPATH=./pymgm:$PYTHONPATH
ERROR: file or directory not found: ./src/pymgm/test/

============================= test session starts ==============================
platform linux -- Python 3.7.17, pytest-7.4.4, pluggy-1.2.0
rootdir: /home/runner/work/s24-meshers/s24-meshers
collected 0 items

============================ no tests ran in 0.00s =============================
Error: Process completed with exit code 4. thsi is the CI output  and  it looks like its coming from this line in the python package action file. I think it might because you now have the test folder as a top level directory so this needs to be updated on line 38 too